src/qt/mwebpegindialog.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2020-2021 The Litecoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_QT_MWEBPEGINDIALOG_H
+#define BITCOIN_QT_MWEBPEGINDIALOG_H
+
+#include <amount.h>
+
+#include <QAbstractButton>
+#include <QDialog>
+
+class PlatformStyle;
+class WalletModel;
+
+namespace Ui {
+    class MWEBPegInDialog;
+}
+
+class MWEBPegInDialog : public QDialog
```

Comment:
Instead of a whole new peg-in dialog, our thought was we would just mark the "Pay To" field as read-only, and change its value to something like: `Peg-In: mweb1t59jsdkjalkwj4t9mmmjelg4ndnvqmntw5dc2gx73tzx3delqqgw5308z98malsvxta4vl9u6dytmmmjelg4ndnvq4t9u0u1245jasldkfaw4ifffoerpql19`

Otherwise, we'll lose access to the ability to label, "subtract fee from amount", "Use available balance", etc.

Reply:
The logic should work like this:
* When the user clicks on Peg-In LTC, the button should stay depressed, and the "Pay To" field should be updated and marked read-only.
* If they decide they don't want to peg-in, they can just click the button again, and it will change to unpressed, and Pay To will be cleared and editable.